_posts/2023-05-02-iniciando-os-estudos.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Iniciando os estudos do zero
+date: 2023-05-02 18:44:00 +/-TTTT
+categories: [Guia]
+tags: [guia, anki, hiragana, katakana]
+author: nicolasanaia
+comments: false
+---
+
+Vou começar sendo sincero e direto ao ponto: eu não acreditava que conseguiria estudar japonês sozinho. Aprender a falar já seria uma tarefa complicada, mas aí surgem os agravantes que assustam os não nativos: **hiragana, katakana e kanji**. Estarei mentindo se disser que nunca tentei aprender os dois primeiros, mas nas vezes em que o fiz, foram com vídeos no Youtube que mais te ensinavam a escrever do que uma técnica pra lembrar em si.
+
+Depois de muito tempo com o **Gustavo Maia**, criador do site [Aprenda Japonês](https://aprendajp.com/), insistindo para que eu tentasse, finalmente resolvi dar uma chance. Vou detalhar um pouco mais o passo a passo de [como comecei](https://aprendajp.com/posts/iniciando-os-estudos/#primeiros-passos) e os [resultados que obtive](https://aprendajp.com/posts/iniciando-os-estudos/#resultados-rápidos) em 2 semanas (até menos).
```

Comment:
se não me engano (tambem nao to com o dev env configurado) para linkar para anchors do mesmo artigo, voce precisa por só a tag

linkar para o proprio site, tipo [Aprenda Japones](http://...) até funciona mas tem um loadzinho
```suggestion
Depois de muito tempo com o **Gustavo Maia**, criador do site [Aprenda Japonês](https://aprendajp.com/), insistindo para que eu tentasse, finalmente resolvi dar uma chance. Vou detalhar um pouco mais o passo a passo de [como comecei](#primeiros-passos) e os [resultados que obtive](#resultados-rápidos) em 2 semanas (até menos).
```

Reply:
pra linkar pra um artigo ou pagina do site, a syntax é:

```markdown
[por que mais um site te ensinando a aprender japonês.]({% link _posts/2022-01-22-por-que-mais-um-site-ensinando-japones?.md %})
```

Reply:
vdd, acho que é isso mesmo
https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown